crates/solvers/src/api/mod.rs

```diff
@@ -59,16 +59,11 @@ async fn solve(
 
         tracing::trace!(?auction);
 
-        let solutions = state
-            .solve(auction)
-            .await
-            .into_iter()
-            .next()
-            .map(|solution| dto::Solutions::from_domain(&[solution]))
-            .unwrap_or_default();
+        let solutions = state.solve(auction).await;
```

Comment:
unrelated

Reply:
I already made a separate PR for this: https://github.com/cowprotocol/services/pull/1799

It should go away once that PR merges and I rebase (but needed to make the test pass).